### PR TITLE
Update update headers in hassio

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change

Part of brand rename.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue: 
- Link to documentation pull request: 

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: hassio/src/snapshots/hassio-snapshots.ts [hunk 1]
```diff
@@ -83,8 +83,8 @@ class HassioSnapshots extends LitElement {
           Create snapshot
         </h1>
         <p class="description">
-          Snapshots allow you to easily backup and restore all data of your
-          Hass.io instance.
+          Snapshots allow you to easily backup and restore all data of your Home
```
Comment (iantrich) [inline hassio/src/snapshots/hassio-snapshots.ts]: Should have access to `hass` here, prefer localize

File: hassio/src/dashboard/hassio-update.ts [hunk 2]
```diff
@@ -64,7 +64,7 @@ export class HassioUpdate extends LitElement {
         </h1>
         <div class="card-group">
           ${this._renderUpdateCard(
-            "Home Assistant",
+            "Home Assistant Core",
```
Comment (iantrich) [inline hassio/src/dashboard/hassio-update.ts]: Should have access to `hass` here, prefer localize

File: hassio/src/snapshots/hassio-snapshots.ts (hunk 1 above)
Comment (iantrich) [reply hassio/src/snapshots/hassio-snapshots.ts]: True for all files touched

**Review by bramkragten — APPROVED** [review]

File: hassio/src/snapshots/hassio-snapshots.ts (hunk 1 above)
Comment (balloob) [reply hassio/src/snapshots/hassio-snapshots.ts]: Yes, but that's out of scope for this PR, which is just about applying the rebrand.

**Pull request merged**